core/src/main/java/ai/timefold/solver/core/impl/heuristic/selector/entity/EntitySelectorFactory.java

```diff
@@ -198,71 +199,65 @@ private EntitySelector<Solution_> applyFiltering(EntitySelector<Solution_> entit
     }
 
     protected void validateSorting(SelectionOrder resolvedSelectionOrder) {
-        if ((config.getSorterManner() != null || config.getSorterComparatorClass() != null
+        var sorterManner = config.getSorterManner();
```

Comment:
Consider applying the same pattern used in multiple places: declare the variable and then check if it is null.